contributors/design-proposals/daemonset-update.md

```diff
@@ -206,6 +244,18 @@ Users can use `kubectl rollout` to monitor DaemonSet updates:
 
 - `kubectl rollout status daemonset/<DaemonSet-Name>`: to see the DaemonSet
   upgrade status 
+- `kubectl rollout history daemonset/<DaemonSet-Name>`: to view the history of
+  DaemonSet updates. We use `PodTemplate` created by DaemonSets to store update
+  history.
+- `kubectl rollout undo daemonset/<DaemonSet-Name>`: to rollback a DaemonSet
+
+#### API
+
+Implement a subresource for DaemonSet history (`daemonsets/foo/history`) that
```

Comment:
Can we move this into a separate proposal since a history subresource is not needed for upgrades and it's something we'll need to do for all the workload APIs?